ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/platform.j2

```diff
@@ -4,6 +4,48 @@
 {%     if platform.trident.forwarding_table_partition is arista.avd.defined %}
 platform trident forwarding-table partition {{ platform.trident.forwarding_table_partition }}
 {%     endif %}
+{%     if platform.trident.mmu.active_profile is arista.avd.defined %}
+platform trident mmu queue profile {{ platform.trident.mmu.active_profile }} apply
+{%     endif %}
+{%     for profile in platform.trident.mmu.queue_profiles | arista.avd.natural_sort('name') %}
+!
+platform trident mmu queue profile {{ profile.name }}
+{%         for queue in profile.unicast_queues | arista.avd.natural_sort('id') %}
+{%             if queue.reserved is arista.avd.defined %}
+{%                 set reserved_cli = "egress unicast queue " ~ queue.id ~ " reserved" %}
+{%                 if queue.unit is arista.avd.defined %}
+{%                     set reserved_cli = reserved_cli ~ " " ~ queue.unit %}
+{%                 endif %}
+{%                 set reserved_cli = reserved_cli ~ " " ~ queue.reserved %}
+    {{ reserved_cli }}
+{%             endif %}
+{%             if queue.threshold is arista.avd.defined %}
+    egress unicast queue {{ queue.id }} threshold {{ queue.threshold }}
+{%             endif %}
+{%             if queue.drop is arista.avd.defined %}
+    egress unicast queue {{ queue.id }} drop-precedence {{ queue.drop.precedence }} drop-threshold {{ queue.drop.threshold }}
+{%             endif %}
+{%         endfor %}
+{%         for queue in profile.multicast_queues | arista.avd.natural_sort('id') %}
+{%             if queue.reserved is arista.avd.defined %}
+{%                 set reserved_cli = "egress multicast queue " ~ queue.id ~ " reserved" %}
+{%                 if queue.unit is arista.avd.defined %}
+{%                     set reserved_cli = reserved_cli ~ " " ~ queue.unit %}
+{%                 endif %}
+{%                 set reserved_cli = reserved_cli ~ " " ~ queue.reserved %}
+    {{ reserved_cli }}
+{%             endif %}
+{%             if queue.threshold is arista.avd.defined %}
+    egress multicast queue {{ queue.id }} threshold {{ queue.threshold }}
+{%             endif %}
+{%             if queue.drop is arista.avd.defined %}
+    egress multicast queue {{ queue.id }} drop-precedence {{ queue.drop.precedence }} drop-threshold {{ queue.drop.threshold }}
+{%             endif %}
+{%         endfor %}
+{%         if loop.last %}
+!
+{%         endif %}
```

Comment:
We normally never insert these at the _end_ of a section. I think it might be missing from the following section, which causes the issue I commented on.

Reply:
understood, removed from the template with the refactor.